release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -89,6 +89,24 @@ For `console` and `downloads` routes, custom routes functionally is now implemen
 [id="ocp-4-8-networking"]
 === Networking
 
+[id="ocp-4-8-network-policy-host-network-ingress-controllers"]
+==== Network policy supports selecting host network Ingress Controllers
+
+When using the OpenShift SDN or OVN-Kubernetes cluster network providers, you can select traffic from Ingress Controllers in a network policy rule regardless of whether an Ingress Controller runs on the cluster network or the host network.
+In a network policy rule, the `policy-group.network.openshift.io/ingress: ""` namespace selector always matches traffic from an Ingress Controller.
```

Comment:
I don't know about network policies. _ namespace selector always matches traffic_ Is it that the user must configure the parameter to always match or is there some magic in the background that ensures this?

Reply:
Is the `policy-group.network.openshift.io/ingress:` the same as the `network.openshift.io/policy-group`? Just wondering if  `policy-group.network.openshift.io/ingress` is a new thing.

Reply:
@mburke5678, the policy group label format changed, although I'm not sure why. The behavior is otherwise identical.

Reply:
@mburke5678, this feature introduces some magic so that regardless of whether an ingress controller lives on the host network or the cluster network, a network policy can be crafted to allow incoming traffic from it.

But perhaps the wording is off, and I can just omit _always_, because why wouldn't it? And then the next para describes when previously it did not match traffic.

Reply:
> the policy group label format changed, although I'm not sure why

You can only have one label with a given name (eg, `network.openshift.io/policy-group`), so using names like that meant that any given namespace could only be associated with a single "policy group", and while developing this feature we realized that that's a bug. (In particular, in some cases a particular namespace needs to be part of both the "ingress" and "host-network" policy groups.) So we switched to using a distinct label name for each policy group.

For backward compatibility, the old way is still supported in all of the cases where it used to be supported. So for the "ingress" policy group, you can match against either `network.openshift.io/policy-group: ingress` or `policy-group.network.openshift.io/ingress: ""`. But for the "host network" policy group, you have to use the new-style name `policy-group.network.openshift.io/host-network: ""`